CONTRIBUTING.md

```diff
@@ -43,6 +43,29 @@ as follows:
 
 ## Filenames and folder structure
 
+Please use snake case for file names. Our documentation follows pitchfork for
```

Comment:
Please separate these updates to the CONTRIBUTING.md file into a separate PR. This is a separate update from OpenMP.

Reply:
I have made no changes to the CONTRIBUTING.md file, somehow it showed conflict so I just resolved them.

Reply:
This was a merge conflict. Fixed in https://github.com/RadeonOpenCompute/ROCm/pull/2404/commits/f58a6b55506ed433407b0c8783c7d6c85217b207

The content is already in `docs/contribute/feedback.md`.